studio/schemas/documents/leaderCard.js

```diff
@@ -0,0 +1,25 @@
+export default {
+    name: 'leaderCard',
+    title: 'Leader Card',
+    type: 'document',
+    fields: [
+        {
+            name: 'name',
+            title: 'Name',
+            type: 'string',
+        },
+        {
+            name: 'title',
+            title: 'Title',
+            type: 'string',
+        },
+        {
+            name: 'image',
+            title: 'Image',
+            type: 'mainImage',
+            options: {
+                hotspot: true,
+            },
```

Comment:
I don't think you need this if it's defined in the `mainImage` object

Reply:
@asasmith updated